microcosm_pubsub/daemon.py

```diff
@@ -86,8 +86,8 @@ def __call__(self, graph):
         Implement daemon by sinking messages from the consumer to a dispatcher function.
 
         """
-        result = graph.sqs_message_dispatcher.handle_batch(self.bound_handlers)
-        if not result.message_count:
+        results = graph.sqs_message_dispatcher.handle_batch(self.bound_handlers)
```

Comment:
Instead of return a structure with a `message_count` (which is unused), we return a list of results (with data we should be able to use for metrics)